### PR TITLE
Fixes to README

### PR DESCRIPTION
**Pull request by anenadic**
Fixed some inconsistencies in README after adding more sections to the lesson, course length and teaching sections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,20 +19,20 @@ development-related projects are now becoming larger and more complex and they n
...
-process in a team, starting from an existing piece of software.
-
-The software project template repository used for this lesson is available at:
-<https://github.com/carpentries-incubator/python-intermediate-inflammation>.
-
 ### Lesson Status
-The lesson is currently in beta - it has been tested several times with 
-different cohorts by the lesson authors as well as independently by people not involved in the course development
+
+The lesson is currently in beta - it has been over 10 times times with
```
Comment (douglowe) [inline README.md:24]: ...it has been *run* over 10 times...

File: README.md [hunk 2]
```diff
@@ -17,20 +19,20 @@ development-related projects are now becoming larger and more complex and they n
...
-The lesson is currently in beta - it has been tested several times with 
-different cohorts by the lesson authors as well as independently by people not involved in the course development
+
+The lesson is currently in beta - it has been over 10 times times with 
+different cohorts by the lesson authors as well as independently by people not directly involved in the lesson development
 and is in a good state to be reused and taught by others.
 
 ## Teaching the Lesson
+
+The lesson has 5 sections that can be delivered (approximately) over 5 half-days. The lesson is suitable either for
```
Comment (douglowe) [inline README.md:30]: Should we say 5 half-days, or 5 two-third days? (as I think some of the sections do take longer than a hlaf-day).

**Review by douglowe — CHANGES_REQUESTED** [review]
Mostly looks good to me, just one missing word, and I wonder if we should angle for 2/3 days, not 1/2 days?

File: README.md (hunk 2 above)
Comment (anenadic) [reply README.md:30]: Yes, I was wondering how to indicate the time needed most accurately. Perhaps we could just say 5 days?

File: README.md (hunk 1 above)
Comment (anenadic) [inline README.md:24]:
```suggestion
The lesson is currently in beta - it has been run over 10 times times with 
```

File: README.md (hunk 2 above)
Comment (anenadic) [inline README.md:30]:
```suggestion
The lesson has 5 sections; each section can be delivered or worked through in self-learning mode over a half a day to a day depending on the pace (we would recommend setting aside 1 day per section to make sure the delivery is not rushed). The lesson is suitable either for 
```

Comment (douglowe) [reply README.md:30]: I'm still not 100% certain on this. There's some unevenness in the length of time needed for each of the 5 sections take to teach. When teaching in person we found that we over ran for several of the earlier, coding, lessons. But regained time when teaching the later theoretical lessons.

Perhaps play safe, and suggest that teachers put aside 5 days for the course. But state that each section will take between half a day, and a full day, to teach.

Comment (anenadic) [reply README.md:30]: How about this for an estimate in time needed @douglowe ?

Comment (douglowe) [reply README.md:30]: that works for me :)

**Review by douglowe — APPROVED** [review]

**Pull request merged**